spec/models/evidence_summary_spec.rb

```diff
@@ -285,4 +285,38 @@
       end
     end
   end
+
+  describe '#recent?' do
+    let(:blocks) do
+      [
+        {
+          'identifier' => 'content', 'content' => '<p>Content</p>'
+        }.merge(created_at)
+      ]
+    end
+
+    context 'when the date is after 90 days ago' do
+      let(:created_at) { { 'created_at' => (Time.current - 5.days).to_s } }
+
+      it 'returns true' do
+        expect(evidence_summary.recent?).to be_truthy
+      end
+    end
+
+    context 'when the date is before 90 days ago' do
```

Comment:
I don't know if I understood wrong but is it not the other way around? s/before/after/ ? The same with above context.

Reply:
I think you may have misunderstood. Would it be more clear if I used 

`later than 90 days ago`  in place of `after 90 days ago` 
and 
`earlier than 90 days ago` in place of `before 90 days ago`

Reply:
Ok I got it now. Yes it will be more clear if you use your suggestion above. 🙏